hdr_schemata/examples/GWDM/1.0/example.json

```diff
@@ -0,0 +1,126 @@
+{
+    "required": {
+        "gatewayId": "1234",
+        "gatewayPid": "5124f2",
+        "issued": "2020-08-05T14:35:59Z",
+        "modified": "2021-01-28T14:15:46Z",
+        "revisions": [
```

Comment:
Only a question really, but do we need to include a `latest` here?

Reply:
Quite possibly. I noticed quite a few things yesterday when I was going through the model yesterday and I created:
https://hdruk.atlassian.net/browse/GAT-3015

I believe Stephen/Ping have found some things that might be missing and we may need to add to a GWDM 1.1 , so will add this to the list too :)